modules/tinymce/src/plugins/charmap/test/ts/module/Helpers.ts

```diff
@@ -0,0 +1,12 @@
+import { SugarElement } from '@ephox/sugar';
+
+// TODO: Move into shared library
```

Comment:
Should we open a JIRA for this (if there isn't one already)?

Reply:
I mentioned in one of the other PR's I plan on doing that after (ie scoping through and creating JIRAs for all things we've identified that need fixing). That'll let me group them better.

Reply:
Okay sounds good 👍